packages/mgt/src/components/mgt-person-card/sections/mgt-person-card-profile/graph.profile.ts

```diff
@@ -0,0 +1,607 @@
+  /**
+   * The month and year the user graduated or completed the activity.
+   *
+   * @type {Date}
+   * @memberof IEducationalActivity
+   */
+  completionMonthYear: Date;
+  /**
+   * The month and year the user completed the educational activity referenced.
+   *
+   * @type {Date}
+   * @memberof IEducationalActivity
+   */
+  endMonthYear: Date;
+  /**
+   * Contains details of the institution studied at.
+   *
+   * @type {IInstitutionData}
+   * @memberof IEducationalActivity
+   */
+  institution: IInstitutionData;
+  /**
+   * Contains extended information about the program or course.
+   *
+   * @type {IEducationalActivityDetail}
+   * @memberof IEducationalActivity
+   */
+  program: IEducationalActivityDetail;
+  /**
+   * The month and year the user commenced the activity referenced.
+   *
+   * @type {Date}
+   * @memberof IEducationalActivity
+   */
+  startMonthYear: Date;
+}
+
+/**
+ * Represents descriptive properties of a person in a tenant.
+ * These properties are surfaced in shared people experiences across Microsoft 365
+ * and third-party services and experiences via Microsoft Graph.
+ *
+ * @interface IProfile
+ */
+export interface IProfile {
+  // tslint:disable-next-line: completed-docs
+  id: string;
+  /**
+   * Represents the details of meaningful dates associated with a person.
+   *
+   * @type {IPersonAnniversary[]}
+   * @memberof IProfile
+   */
+  anniversaries: IPersonAnniversary[];
+  /**
+   * Represents data that a user has supplied related to undergraduate, graduate, postgraduate or other educational activities.
+   *
+   * @type {IEducationActivity[]}
+   * @memberof IProfile
+   */
+  educationalActivities: IEducationalActivity[];
+  /**
+   * Provides detailed information about interests the user has associated with themselves in various services.
+   *
+   * @type {IPersonInterest[]}
+   * @memberof IProfile
+   */
+  interests: IPersonInterest[];
+  /**
+   * Represents detailed information about languages that a user has added to their profile.
+   *
+   * @type {ILanguageProficiency[]}
+   * @memberof IProfile
+   */
+  languages: ILanguageProficiency[];
+  /**
+   * Represents detailed information about work positions associated with a user's profile.
+   *
+   * @type {IWorkPosition[]}
+   * @memberof IProfile
+   */
+  positions: IWorkPosition[];
+  /**
+   * Represents detailed information about skills associated with a user in various services.
+   *
+   * @type {ISkillProficiency[]}
+   * @memberof IProfile
+   */
+  skills: ISkillProficiency[];
+}
+
+/**
+ * TODO: Figure out the correct graph call
+ *
+ * @export
+ * @param {IGraph} graph
+ * @param {string} userId
+ * @returns {IProfile}
+ */
+export async function getProfile(graph: IGraph, userId: string): Promise<IProfile> {
+  const profile = await graph.api(`/users/${userId}/profile`).get();
+  //injectDummyData(profile);
+  return profile;
+}
+
+// tslint:disable
+function injectDummyData(profile) {
```

Comment:
Does this section need to be removed? Or maybe saved to a different place as part of a story?

Reply:
Removed